RELEASE_NOTES.md

```diff
@@ -1,5 +1,13 @@
 # Release Notes
 
+## 5.3.1
+### Bugfixes
+- Fixed a bug where the `--ignition-path` option to `podman machine init` would prevent creation of necessary files for the VM, rendering it unusable ([#23544](https://github.com/containers/podman/issues/23544)).
+- Fixed a bug where rootless containers using the `bridge` networking mode would be unable to start due to a panic caused by a nil pointer dereference ([#24566](https://github.com/containers/podman/issues/24566)).
+- Fixed a bug where Podman containers would try to set increased rlimits when started in a user namespace, rendering containers unable to start ([#24547](https://github.com/containers/podman/pull/24547)).
```

Comment:
This links a PR not the issue https://github.com/containers/podman/issues/24508

Reply:
Fixed